CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
 
 #### _**Note**_: Breaking changes
 
+### 6.4.0
+
+#### New features
+* Fix #4694: [java-generator] Option to override the package name of the generated code.
+
```

Comment:
This should be moved to the SNAPSHOT section, I guess that since we released 6.3.1 while you were working on this you didn't notice.